### PR TITLE
Add TektonCD CLI CI

### PR DESCRIPTION
**Pull request by chmouel**

### CODE REVIEW

File: ci-operator/config/openshift/tektoncd-cli/openshift-tektoncd-cli-release-next.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+tag_specification:
+  name: '4.0'
+  namespace: ocp
+promotion:
+  cluster: https://api.ci.openshift.org
```
Comment (petr-muller) [inline ci-operator/config/openshift/tektoncd-cli/openshift-tektoncd-cli-release-next.yaml]: Cluster is not needed to specify.

File: ci-operator/config/openshift/tektoncd-cli/openshift-tektoncd-cli-release-next.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+tag_specification:
+  name: '4.0'
```
Comment (petr-muller) [inline ci-operator/config/openshift/tektoncd-cli/openshift-tektoncd-cli-release-next.yaml]: I think you should at least use 4.1 - we haven't even released anything 4.0 AFAIK

File: ci-operator/config/openshift/tektoncd-cli/openshift-tektoncd-cli-release-next.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  base:
+    name: '4.0'
+    namespace: ocp
+    tag: base
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/Dockerfile
+canonical_go_repository: github.com/tektoncd/cli
+binary_build_commands: make
+promotion:
```
Comment (petr-muller) [inline ci-operator/config/openshift/tektoncd-cli/openshift-tektoncd-cli-release-next.yaml]: Huh second `promotion`

Comment (chmouel) [reply ci-operator/config/openshift/tektoncd-cli/openshift-tektoncd-cli-release-next.yaml]: Ah thanks, I mostly copied things from https://github.com/openshift/release/blob/master/ci-operator/config/openshift/tektoncd-pipeline/openshift-tektoncd-pipeline-release-next.yaml so I guess i'll send a cleanup patch later on 👍

**Comment by chmouel** [comment]
🎉 

image

**Comment by chmouel** [comment]
/retest

Using clean  `release-next` branch generated with `./openshift/release/update-to-head.sh`

**Pull request merged**